### PR TITLE
fix bugs for trainee scroll and dup message

### PR DESCRIPTION
**Pull request by ksungreborn**
Fix trainee dropdown and duplicate message when internet access approved

### CODE REVIEW

**Comment by billyeh** [comment]
Hopefully that's better. The old server also allows, like, GTCA teammates, but do we have an equivalent for that on the new one?

File: ap/web_access/forms.py [hunk 1]
```diff
@@ -35,7 +36,14 @@ class Meta:
 
 class EShepherdingRequest(forms.Form):
     active_trainees = Trainee.objects.select_related().filter(is_active=True)
```
Comment (jontey) [inline ap/web_access/forms.py]: I don't think you need this line since Trainee Object returns active trainees by default.

**Comment by jontey** [comment]
Yeah.. Gospel trips have not been added yet

File: ap/web_access/forms.py (hunk 1 above)
Comment (billyeh) [inline ap/web_access/forms.py]: Yeah, you're right.

**Comment by billyeh** [comment]
I accidentally created the branch for #454 off this branch, so the majority of the changes are already merged into dev. This is basically just a pull request for some one-line code cleanup now.

**Pull request merged**